src/proto/FunctionRpc.proto

```diff
@@ -337,6 +337,9 @@ message RpcFunctionMetadata {
   // A flag indicating if managed dependency is enabled or not
   bool managed_dependency_enabled = 14;
 
+  // Retry policy of the function
+  RetryPolicy retry = 15;
```

Comment:
Have you by chance tried using this in the host yet? I'm curious if we want this to be an explicit type or if we want it to be a json-serialized string. For example, in the case of bindings we have both `raw_bindings` and `bindings`, so I wasn't sure if we needed `raw_retry`, `retry`, or both. I feel like once this is implemented in the host we should know the answer pretty quickly.

- [Here](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script/Description/Workers/Rpc/RawFunctionMetadata.cs#L15) is where I see the retry options referenced as a raw string
- [Here](https://github.com/Azure/azure-functions-host/blob/dev/src/WebJobs.Script.Abstractions/Description/FunctionMetadata.cs#L48) is where I see the retry options referenced as an explicit type

Reply:
I haven't tried it with the host yet since the host implementation is not done yet. I'll let someone from the host team confirm what data type the host code expects.